configurator/src/cv.rs

```diff
@@ -159,6 +161,7 @@ impl Drop for Display {
 pub fn find_light_from_diff(
     base_picture: &Picture,
     led_picture: &Picture,
+    output_dir: Option<PathBuf>,
```

Comment:
I'm not sure I like this method taking an `output_dir`, but I don't have a better idea...